tests/test_sampling.py

```diff
@@ -63,6 +63,15 @@ def test_rles_to_points():
     np.testing.assert_array_equal(points, expected)
 
 
+def test_sample_supervoxels():
+    """Test that the label at a point is correctly returned."""
+    server = "https://hemibrain-dvid.janelia.org"
+    uuid = "15aee239283143c08b827177ebee01b3"
+    instance = "segmentation"
+    body_id = 1137590955
+    supervoxels = sample_supervoxels(server, uuid, instance, body_id)
+    assert supervoxels.shape == (4911,)
```

Comment:
This doesn't test content and also relies on that DVID server being around permanently. Suggest getting the actual result from server and adding it to mock server. In this case, you're just testing that it handles a string of List[int] appropriately.